server/enterprise/metrics/metrics.go

```diff
@@ -1244,6 +1249,42 @@ func New(ps *platform.PlatformService, driver, dataSource string) *MetricsInterf
 	)
 	m.Registry.MustRegister(m.ClientGlobalThreadsLoadDuration)
 
+	m.MobileClientLoadDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Namespace: MetricsNamespace,
+			Subsystem: MetricsSubsystemClientsMobileApp,
+			Name:      "mobile_load",
+			Help:      "Duration of the time taken from when a user opens the app and the app finally loads all relevant information (milliseconds)",
```

Comment:
I'm not sure if this is a typo copied from above (`global_threads_load` should be in seconds like the other web app metrics), but these should also be measured in seconds because that's the standard for Prometheus metrics. I ran into the same thing with the web app metrics too

Reply:
Yeah, I wrote this down when I first implemented it, and when we decided to go with seconds, I forgot to update it.